src/Form/FormAnalyzer.js

```diff
@@ -17,11 +17,11 @@ class FormAnalyzer {
     }
 
     get isLogin () {
-        return this.autofillSignal <= 0
+        return this.autofillSignal < 0
     }
 
     get isSignup () {
-        return this.autofillSignal > 0
+        return this.autofillSignal >= 0
```

Comment:
I flipped these. This means we're now leaning more on showing Dax when possible, preferring to avoid false negatives. In tests it didn't have a negative impact, and it will be improved when we introduce identities autofill.